docs/source/development/sources.rst

```diff
@@ -38,6 +38,34 @@ Fedora
     systemctl enable rabbitmq-server
     systemctl restart rabbitmq-server
 
+CentOS
+-----------
```

Comment:
```suggestion
CentOS/RHEL
-----------
```

:+1: 

We also can remove the Fedora section above, as that was really meant yum-like systems.